lazyflow/operators/generic.py

```diff
@@ -532,24 +534,20 @@ def setupOutputs(self):
         self.Output.meta.drange = (0, 1)
 
     def execute(self, slot, subindex, roi, result):
-        key = roi.toSlice()
-        data = self.inputs["Input"][key[:-1] + (slice(None),)].wait()
+        *key, c = roi.toSlice()
 
-        dm = numpy.max(data, axis=data.ndim - 1, keepdims=True)
-        res = numpy.zeros(data.shape, numpy.uint8)
-        numpy.equal(data, dm, out=res)
+        n_channels_requested = c.stop - c.start
```

Comment:
Is it possible for our ROIs to have `c.start is None` or `c.stop is None`?

Reply:
thx to your type annotations I can confidently say: no :)
```Python
def roiToSlice(start: Sequence[numbers.Integral], stop: Sequence[numbers.Integral])
```